lua/type-fmt.lua

```diff
@@ -27,7 +27,7 @@ end
 
 local function get_otf_client(bufnr)
 	local result
-	for _, client in ipairs(vim.lsp.get_active_clients({ bufnr = bufnr })) do
+	for _, client in ipairs(vim.lsp.get_clients({ bufnr = bufnr })) do
```

Comment:
It would be better to keep compatible with nvim<0.10
```lua
local lsp_get_clients = vim.lsp.get_clients or vim.lsp.get_active_clients 
for _, client in ipairs(lsp_get_clients{ bufnr = bufnr })) do
```

Reply:
done :+1: